docs/ReleaseProcess.md

```diff
@@ -350,3 +308,99 @@ git pull
 
 [contributing-to-a-project]: https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#Commit-Guidelines
 [chap]: https://github.com/input-output-hk/cardano-haskell-packages
+
+# The first time we had to break this release process
+
+On the first half of April 2023, we were asked to get a new Consensus release
+from current `master` in order for it to be integrated into the node. However
```

Comment:
Maybe `ourodoros-network@master` or something like that, and adding a footnote that mentions that the code was living in that repo at that time?